src/libfuzzer/libfuzzer_mutator.cc

```diff
@@ -22,8 +22,33 @@
 #include "port/protobuf.h"
 #include "src/mutator.h"
 
+#ifdef _MSC_VER
```

Comment:
with this ifdef WIN_WEAK_ALIAS make not much sense, you can keep just __pragma

I would prefere have common code with equivalent of SANITIZER_INTERFACE_WEAK_DEF
and it will be expanded into platform specific stuff

Reply:
Done.